src/broker/CMakeLists.txt

```diff
@@ -1,20 +1,14 @@
-include(ZeekSubdir)
-
-include_directories(BEFORE
-                    ${CMAKE_CURRENT_SOURCE_DIR}
-                    ${CMAKE_CURRENT_BINARY_DIR}
+zeek_add_subdir_library(
+    comm
```

Comment:
Should we rename this library to `broker` at this point?

Reply:
Will do.